docs/building-apps.md

```diff
@@ -218,6 +219,25 @@ my-app/
   webos-meta/
 ```
 
+## Caching
+
+For supporting better [`Caching`](https://webpack.js.org/guides/caching/), Enact CLI provides `--content-hash` option to add a unique hash to each output file name based on the content of an asset.
+
+Building With this option should produce the following output:
+
+```none
+        1.11 MB         dist/main.1983e557b9a705c83e72.js
+        199.85 kB       dist/main.2088c66150ab73b27793.css
+```
+
+When the content changes, the output file name will change as well.
+```none
+        1.11 MB         dist/main.7544f55b64439c8d0f0e.js
+        199.85 kB       dist/main.2088c66150ab73b27793.css
+```
+
+> **NOTE** The filename `main.*.js` will be changed after bundling, without changes. This is because webpack includes certain boilerplate, specifically the runtime and manifest, in the entry chunk.
```

Comment:
```suggestion
> **NOTE** The filename `main.*.js` will be changed after another building, even without making any changes. This is because webpack includes certain boilerplate, specifically the runtime and manifest, in the entry chunk.
```